oneflow/core/ep/include/device_manager.h

```diff
@@ -40,6 +41,11 @@ class DeviceManager {
   virtual size_t GetDeviceCount() = 0;
   virtual size_t GetActiveDeviceIndex() = 0;
   virtual void SetActiveDeviceByIndex(size_t device_index) = 0;
+
+  virtual std::shared_ptr<RandomGenerator> CreateRandomGenerator(uint64_t seed,
+                                                                 size_t device_index) {
```

Comment:
```suggestion
  virtual std::shared_ptr<RandomGenerator> CreateRandomGenerator(size_t device_index,
                                                                 uint64_t seed) {
```
这样会不会更好，以及这个接口是不是要求必须实现

Reply:
因为燧原那边适配应该没有实现Generator，所以我这里暂时提供了一个默认实现，可以改成必须实现，燧原适配那边改一下代码。device_index和seed的顺序应该还好，暂时不改了。